firmware/port.c

```diff
@@ -140,7 +152,7 @@ void port_disable(PortData* p) {
 
 void port_send_status(PortData* p, u8 d) {
     if (p->reply_len >= BRIDGE_BUF_SIZE) {
-        bridge_disable_chan(p->chan);
+        port_error(p);
```

Comment:
Oops, included the wrong version of this patch. `port_error` should call `bridge_disable_chan` on T2, so as to not change the current behavior. Just meant to abstract it out into a function to avoid merge conflicts going back and forth because starfish doesn't have `bridge_*`.